### DISCUSSION TITLE
Release schedules and features?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3764] [General] [answered]
Are there any documents available for Xournalpp that outline when releases are planned, and what fixes/features they are likely to contain?

**Comment by Technius** [comment, marked as answer]
There is no concrete roadmap document (yet), but completed/planned features and bugfixes are tracked by our milestones: https://github.com/xournalpp/xournalpp/milestones

**Reply by NelkonParker** [reply]
Thanks. That's exactly what I was looking for.
